examples/test-cra/package.json

```diff
@@ -17,19 +17,19 @@
     "react-dom": "^15.5.4"
   },
   "devDependencies": {
-    "@storybook/addon-actions": "file:../../addons/actions",
-    "@storybook/addon-centered": "file:../../addons/centered",
-    "@storybook/addon-info": "file:../../addons/info",
-    "@storybook/addon-knobs": "file:../../addons/knobs",
-    "@storybook/addon-links": "file:../../addons/links",
-    "@storybook/addon-notes": "file:../../addons/notes",
-    "@storybook/addon-options": "file:../../addons/options",
-    "@storybook/addon-storyshots": "file:../../addons/storyshots",
-    "@storybook/addons": "file:../../lib/addons",
-    "@storybook/channel-postmessage": "file:../../lib/channel-postmessage",
-    "@storybook/channels": "file:../../lib/channels",
-    "@storybook/react": "file:../../app/react",
-    "@storybook/ui": "file:../../lib/ui",
+    "@storybook/addon-actions": "^3.0.1",
```

Comment:
This removes the purpose of `test-cra`

I at one point set it up as an alternative if bootstrapping wouldn't work.

Nowadays bootstrapping is relatively fast-ish (compared to what is was) and it's reliable (at least for me & on CI).

I think we can discontinue this example now, and stop using file dependencies. @tmeasday I'd like your thoughts on this?